composer.json

```diff
@@ -19,7 +19,8 @@
     },
     "require-dev": {
         "phpunit/phpunit": "^4.5 || ^5.4",
-        "guzzlehttp/psr7": "^1.3"
+        "guzzlehttp/psr7": "^1.3",
+        "php-http/message": "^1.0"
```

Comment:
okay, makes sense. then we have to make sure the installation
instructions are clear.